glaciers/glaciers.ipynb

```diff
@@ -37,13 +27,13 @@
     "import numpy as np\n",
     "import pandas as pd\n",
     "import holoviews as hv\n",
-    "import datashader as ds\n",
     "import panel as pn\n",
+    "import hvplot.pandas # noqa\n",
     "\n",
     "from colorcet import bmy\n",
-    "from holoviews.operation.datashader import rasterize, datashade\n",
+    "from holoviews.util.transform import lon_lat_to_easting_northing as ll_en\n",
     "\n",
-    "hv.extension('bokeh')"
+    "pn.extension('tabulator', template='fast')"
```

Comment:
Given that you manually instantiate the template at the end of the notebook, you shouldn't declare it upfront. This notebook/app isn't using the `Tabulator` widget so there's no need to load it. In fact, I don't think calling `pn.extension()` is required at all in this notebook since it's implicitly called by `import hvplot.pandas`. I might be wrong though so you could just try and see what happens.

Reply:
Ok @philippjfr just confirmed you shouldn't need to call `pn.extension()` in this case.